package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
Er, why is a `package-lock.json` being added to the repo?  Should this be removed, since we use yarn instead of npm?

Reply:
gah!